search-app/resources/templates/search-provider-tag-landing-links.html

```diff
@@ -1,14 +1,39 @@
 {% extends "templates/search-base.html" %}
 
-{% block title %}{{ app-title }} - Landing Pages for the {{ provider-name}} {{ tag-name }} Collections{% endblock %}
+{% block title %}{{ app-title }} - Landing Pages for the {{ provider-id}} {{ tag-name }} Collections{% endblock %}
```

Comment:
"the" can be dropped from this line. For example, it should read "Landing pages for NSIDC Collections". I'd also be weary of using provider-id. I think Provider Name would make more sense to end users not familiar with how we refer to providers internally.